### DISCUSSION TITLE
BirdNet - Live Stream Demo documentation?

### DISCUSSION THREAD

**Discussion by evolsoulx** [birdnet-team/BirdNET-Analyzer#290] [Q&A]
https://birdnet.cornell.edu/live/

is there any documentation on how to make this but for any streaming audio file?

I'd really like to build something out for crittercam streamers (see mine here: https://twitch.tv/squirrelville) to be able to easily use; feed the audio from cameras to that page, and let them browser source it onto the screen and customize. It could be really, really cool. I just can't seem to be able to recreate that page. 

I don't have a raspberry pi, i tried getting it all set up in a VM but somethings off, i can't find the IP from the VM. Dunno.

**Comment by tphakala** [comment]
That online stream analyzer is not part of BirdNET-Analyzer application available here, it is completely independent application. But I don't quite follow what you are looking for, I don't see how that online analyzer would fit web streams?

If you are looking for BirdNET analyzer based solution which identifies birds from web cam streams and would like to overlay that data to OBS streams then my [BirdNET-Go application](https://github.com/tphakala/birdnet-go) is solution for that, you can see it in use on my birdfeeder stream at https://www.youtube.com/watch?v=tyTbrohFQcw.

**Comment by evolsoulx** [comment]
This is exactly what I’ve been looking for! Oh my god I’m not in front of a laptop but I am SO excited.

I see you’re looking for front end guys, that’s why I’ve been struggling here; I’m more of a front end guy, and just getting into GitHub stuff. I think I can make a branch and make edits yeah? But tomorrow morning I’m 100% implementing this. Thank you so much!!!!!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
